### PR TITLE
Feature/contestant selector

### PR DESCRIPTION
**Pull request by jjm3x3**
Here is the main code necessary to create a contestant selector. More contestants to be added next.

_credits: I needed to shop around a bit and ask for a few different folks opinion eventually I settled on adopting [this solution from SO](https://stackoverflow.com/a/76766207)_

### CODE REVIEW

**Review by CAnthonySiever — APPROVED** [review]
Looks good to me 👍

**Review by internette — CHANGES_REQUESTED** [review]
Testing?

**Comment by jjm3x3** [comment]
> Testing?

What about it? What testing should happen here?

File: app/scoring/page.tsx [hunk 1]
```diff
@@ -57,16 +58,16 @@ export default async function Scoring() {
...
     const contestantRoundScores: number[] = generateContestantRoundScores(currentSelectedContestantTeamsList, numberOfRounds)
 
+    const listOfContestantRoundLists = [{
+            key: "Jacob",
+            content: <ContestantRoundList perfectRoundScores={roundScores} contestantRoundScores={contestantRoundScores} perfectTeamList={reverseTeamsList} contestantTeamList={currentSelectedContestantTeamsList}/>
+    }]
+
     return (
         <div>
-            <h1 className="text-2xl text-center">Current Scoring for {currentSelectedContestant}</h1>
```
Comment (jjm3x3) [inline app/scoring/page.tsx:62]: I should remove `currentSelectedContestant` from the code since this is no longer used after this.

**Review by internette — APPROVED** [review]

**Review by jjm3x3 — COMMENTED** [review]
What is my comment 👀 
image

File: jest.config.mjs [hunk 2]
```diff
@@ -1,18 +1,18 @@
-import nextJest from 'next/jest.js'
- 
+import nextJest from "next/jest.js";
+
+/** @type {import('jest').Config} */
```
[the remark this answers is not in the document]
Comment (jjm3x3) [reply jest.config.mjs]: According to @internette it is not necessary!

**Pull request merged**